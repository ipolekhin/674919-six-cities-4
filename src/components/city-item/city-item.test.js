import React from "react";
import renderer from "react-test-renderer";
import CityItem from "./city-item.jsx";

const city = `Amsterdam`;
const currentCity = `Amsterdam`;
const onCityClick = () => {};
const onActiveItemChange = () => {};

it(`Render CityItem`, () => {
  const tree = renderer
    .create(
        <CityItem
          city = {city}
          currentCity = {currentCity}
          onActiveItemChange = {onActiveItemChange}
          onCityClick = {onCityClick}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
