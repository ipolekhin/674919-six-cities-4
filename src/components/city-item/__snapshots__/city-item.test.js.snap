// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render CityItem 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
    onClick={[Function]}
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;
